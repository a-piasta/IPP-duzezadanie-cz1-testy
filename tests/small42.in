repairRoad;mnvhioagfedztlabbfqsmvllnz;cbohgsnomuablwlbrruky;-13
extendRoute;50;uthgrnzewdfkmodzcrv
newRoute;817;mnvhioagfedztlabbfqsmvllnz;tqzhvamnvqietoool
newRoute;268;yfbcrozzznbfvfj;wjpterqflivamvesl
extendRoute;817;sghtzdajnk
addRoad;sghtzdajnk;uthgrnzewdfkmodzcrv;28;-15
getRouteDescription;817
addRoad;mnvhioagfedztlabbfqsmvllnz;cbohgsnomuablwlbrruky;7;-7
getRouteDescription;268
newRoute;62;yfbcrozzznbfvfj;wjpterqflivamvesl
getRouteDescription;817
addRoad;cbohgsnomuablwlbrruky;thbekuxeevnyrwhkjiog;14;-5
addRoad;yfbcrozzznbfvfj;thbekuxeevnyrwhkjiog;4;12
getRouteDescription;268
addRoad;sghtzdajnk;mnvhioagfedztlabbfqsmvllnz;15;-14
getRouteDescription;62
removeRoad;mnvhioagfedztlabbfqsmvllnz;cbohgsnomuablwlbrruky
addRoad;sghtzdajnk;mnvhioagfedztlabbfqsmvllnz;29;13
getRouteDescription;817
addRoad;thbekuxeevnyrwhkjiog;sngyipfpphctxsociem;11;4
getRouteDescription;817
addRoad;mnvhioagfedztlabbfqsmvllnz;yfbcrozzznbfvfj;20;-9
addRoad;auubxbspnncacencjofdfeegshln;wjpterqflivamvesl;15;9
addRoad;sngyipfpphctxsociem;tqzhvamnvqietoool;29;-10
getRouteDescription;822
newRoute;194;uthgrnzewdfkmodzcrv;thbekuxeevnyrwhkjiog
newRoute;117;sghtzdajnk;yfbcrozzznbfvfj
repairRoad;cbohgsnomuablwlbrruky;thbekuxeevnyrwhkjiog;-6
extendRoute;817;wjpterqflivamvesl
removeRoad;cbohgsnomuablwlbrruky;thbekuxeevnyrwhkjiog
repairRoad;sghtzdajnk;mnvhioagfedztlabbfqsmvllnz;3
repairRoad;mnvhioagfedztlabbfqsmvllnz;wjpterqflivamvesl;14
extendRoute;50;thbekuxeevnyrwhkjiog
removeRoad;auubxbspnncacencjofdfeegshln;wjpterqflivamvesl
getRouteDescription;194
getRouteDescription;268
removeRoad;sghtzdajnk;mnvhioagfedztlabbfqsmvllnz
getRouteDescription;117
addRoad;wjpterqflivamvesl;sngyipfpphctxsociem;19;-9
repairRoad;sngyipfpphctxsociem;mnvhioagfedztlabbfqsmvllnz;-14
repairRoad;wjpterqflivamvesl;sghtzdajnk;-2
repairRoad;tqzhvamnvqietoool;yfbcrozzznbfvfj;8
extendRoute;817;mnvhioagfedztlabbfqsmvllnz
repairRoad;sngyipfpphctxsociem;yfbcrozzznbfvfj;15
addRoad;sghtzdajnk;auubxbspnncacencjofdfeegshln;10;2
getRouteDescription;817
removeRoad;sghtzdajnk;uthgrnzewdfkmodzcrv
extendRoute;268;thbekuxeevnyrwhkjiog
getRouteDescription;577
getRouteDescription;817
extendRoute;268;uthgrnzewdfkmodzcrv
extendRoute;62;mnvhioagfedztlabbfqsmvllnz
getRouteDescription;62
removeRoad;sghtzdajnk;mnvhioagfedztlabbfqsmvllnz
extendRoute;117;thbekuxeevnyrwhkjiog
getRouteDescription;62
newRoute;497;yfbcrozzznbfvfj;tqzhvamnvqietoool
