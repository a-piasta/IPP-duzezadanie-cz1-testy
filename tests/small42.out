1: NIE
2: NIE
3: NIE
4: NIE
5: NIE
6: TAK
7: 
8: TAK
9: 
10: NIE
11: 
12: TAK
13: TAK
14: 
15: TAK
16: 
17: TAK
18: NIE
19: 
20: TAK
21: 
22: TAK
23: TAK
24: TAK
25: 
26: TAK
27: TAK
28: NIE
29: NIE
30: TAK
31: TAK
32: NIE
33: NIE
34: TAK
35: 194;uthgrnzewdfkmodzcrv;28;-15;sghtzdajnk;15;3;mnvhioagfedztlabbfqsmvllnz;20;-9;yfbcrozzznbfvfj;4;12;thbekuxeevnyrwhkjiog
36: 
37: NIE
38: 117;sghtzdajnk;15;3;mnvhioagfedztlabbfqsmvllnz;20;-9;yfbcrozzznbfvfj
39: TAK
40: NIE
41: NIE
42: NIE
43: NIE
44: NIE
45: TAK
46: 
47: NIE
48: NIE
49: 
50: 
51: NIE
52: NIE
53: 
54: NIE
55: TAK
56: 
57: TAK
